recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
does conda support aliases? or would it just be a duplicate of the same feedstock?
if `conda install asl` should do the same thing as `conda install ampl/mp`, then perhaps we merge this as is and leave the `ampl/mp` alias for another time